google/datalab/contrib/pipeline/_pipeline.py

```diff
@@ -104,6 +107,12 @@ def py(self):
     return Pipeline._imports + default_args + dag_definition + \
         task_definitions + up_steam_statements
 
+  @staticmethod
+  def _get_datetime_expr_str(date_string):
```

Comment:
Why are you returning an expression containing a call to strptime? Can  you instead do the call to strptime here and return the resulting string in quotes?

Reply:
Also, changed this a little bit to work-around yaml.load() returning unaware datetime objects.